client/dcr: Ticket purchase causing panic.

Panic buying a ticket on mainnet on master. Probably related to swapping to the v5 dcrwallet dep.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x38 pc=0x1248550]

goroutine 3676 [running]:
github.com/decred/dcrd/mixing/mixclient.(*Client).Dicemix(0x0, {0x2450d18, 0x4007c23e30}, 0x4000d3c540)
        /home/joe/go/pkg/mod/github.com/decred/dcrd/mixing@v0.5.1-0.20250319155359-2b7d311f4a81/mixclient/client.go:857 +0x30
decred.org/dcrwallet/v5/wallet.(*Wallet).mixedSplit(0x4000886800, {0x2450d18, 0x4007c23e30}, 0x4000d63480, 0x5914a0cdc)
        /home/joe/go/pkg/mod/decred.org/dcrwallet/v5@v5.0.0-20250407130412-4f0acd20d74c/wallet/createtx.go:1079 +0x7c8
decred.org/dcrwallet/v5/wallet.(*Wallet).purchaseTickets(0x4000886800, {0x2450d18, 0x4007c23e30}, {0x19d8851, 0x16}, {0x245a490, 0x40001f6160}, 0x4000d63480)
        /home/joe/go/pkg/mod/decred.org/dcrwallet/v5@v5.0.0-20250407130412-4f0acd20d74c/wallet/createtx.go:1427 +0xf78
decred.org/dcrwallet/v5/wallet.(*Wallet).PurchaseTickets(0x4000886800, {0x244e8f8?, 0x4000592230?}, {0x245a490, 0x40001f6160}, 0x4000d63480)
        /home/joe/go/pkg/mod/decred.org/dcrwallet/v5@v5.0.0-20250407130412-4f0acd20d74c/wallet/wallet.go:1600 +0x98
decred.org/dcrdex/client/asset/dcr.(*spvWallet).PurchaseTickets(0x40000f4840, {0x244e8f8, 0x4000592230}, 0x1, {0x40000f2768, 0x13}, {0x4000ade3c0, 0x2c}, 0x1)
        /home/joe/git/dcrdex/client/asset/dcr/spv.go:1092 +0x354
decred.org/dcrdex/client/asset/dcr.(*ExchangeWallet).runTicketBuyer(0x40004fc908)
        /home/joe/git/dcrdex/client/asset/dcr/dcr.go:5701 +0x6ec
created by decred.org/dcrdex/client/asset/dcr.(*ExchangeWallet).PurchaseTickets in goroutine 3625
        /home/joe/git/dcrdex/client/asset/dcr/dcr.go:5611 +0x33c
```

--- a/client/asset/dcr/spv.go
+++ b/client/asset/dcr/spv.go
@@ -1528,6 +1528,9 @@ func newWalletConfig(db wallet.DB, chainParams *chaincfg.Params, gapLimit uint32
 		RelayFee:        defaultRelayFeePerKb,
 		Params:          chainParams,
 		MixSplitLimit:   defaultMixSplitLimit,
+		// TODO: Wallet should be closed and reopened when turning on
+		// and off mixing, and this should be a variable.
+		MixingEnabled: true,
 	}
 }
 

